specification/security/resource-manager/Microsoft.Security/preview/2019-01-01-preview/networkData.json

```diff
@@ -0,0 +1,408 @@
+          "properties": {
+            "timeRange": {
+              "readOnly": true,
+              "type": "string",
+              "description": "The actual traffic time range"
+            },
+            "TrafficDataState": {
+              "readOnly": true,
+              "type": "string",
+              "description": "The actual traffic state"
+            },
+            "inboundConnectedResources": {
+              "readOnly": true,
+              "type": "object",
+              "description": "The actual traffic inbound details",
+              "properties": {
+                "actualTrafficVolume": {
+                  "readOnly": true,
+                  "type": "string",
+                  "description": "The actual traffic inbound volume"
+                },
+                "connectionsDetails": {
+                  "type": "array",
+                  "items": {
+                    "$ref": "#/definitions/ConnectionDetails"
+                  }
+                }
+              }
+            },
+            "outboundConnectedResources": {
+              "readOnly": true,
+              "type": "object",
+              "description": "The actual traffic outbound details",
+              "properties": {
+                "actualTrafficVolume": {
+                  "readOnly": true,
+                  "type": "string",
+                  "description": "The actual traffic outbound volume"
+                },
+                "connectionsDetails": {
+                  "type": "array",
+                  "items": {
+                    "$ref": "#/definitions/ConnectionDetails"
+                  }
+                }
+              }
+            }
+          }
+        },
+        "unusedPorts": {
+          "readOnly": true,
+          "type": "object",
+          "description": "The unused ports details",
+          "properties": {
+            "inboundConnectedResources": {
+              "readOnly": true,
+              "type": "object",
+              "description": "The unused ports inbound details",
+              "properties": {
+                "unusedTcpPortsSummary": {
+                  "readOnly": true,
+                  "type": "integer",
+                  "description": "The unused inbound tcp ports summary"
+                },
+                "unusedUdpPortsSummary": {
+                  "readOnly": true,
+                  "type": "integer",
+                  "description": "The unused inbound udp ports summary"
+                },
+                "tcpPorts": {
+                  "readOnly": true,
+                  "type": "string",
+                  "description": "The unused inbound tcp ports"
+                },
+                "udpPorts": {
+                  "readOnly": true,
+                  "type": "string",
+                  "description": "The unused inbound udp ports"
+                }
+              }
+            },
+            "outboundConnectedResources": {
```

Comment:
there is a lot of duplicates between the inbound and outbound that might make it harder to maintain
is it possible to have one object with another fied of "connectionType" that will be and "Inbound" "Outbound" enum?